modules/uncertainty/NAMESPACE

```diff
@@ -1,15 +1,15 @@
 # Generated by roxygen2: do not edit by hand
 
+S3method(plot,flux.uncertainty)
+S3method(plot,sensitivities)
+S3method(plot,sensitivity)
```

Comment:
This is a naming issue that we should try to resolve. Every R style guide advises against using dots in names, and especially in exported functions, for this exact reason -- Roxygen interprets common ones as S3 methods.